index.html

```diff
@@ -1,13 +1,999 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I would suggest that "content="IE=edge" should be removed as it is no longer necessary for modern browsers to specify